GoogleCombined/GoogleProject/OverlayController.swift

```diff
@@ -0,0 +1,145 @@
+/* Copyright (c) 2020 Google Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import UIKit
+import GooglePlaces
+import GoogleMaps
+
+class OverlayController {
+    
+    private var overlays = [GMSCircle]()
+    private var lat: Double = 0.0
+    private var long: Double = 0.0
+    
+    // MARK: Methods to get the placeID from a set of coordinates
+    
+    /// Searches an API for the entire data JSON file based on the lat and long values
+    ///
+    /// - Parameter completion: The completion handler.
+    func fetchData(completion: @escaping ([String : Any]?, Error?) -> Void) {
+        let apiKey: String = apikeys.mapsAPI
+        let url =  "https://maps.googleapis.com/maps/api/geocode/json?&latlng=\(lat),\(long)&key="
+        let search = URL(string: url + apiKey)!
+        let task = URLSession.shared.dataTask(with: search) { (data, response, error) in
+            guard let data = data else { return }
+            do {
+                if let array = try JSONSerialization.jsonObject(
+                    with: data,
+                    options: .allowFragments
+                    ) as? [String : Any] {
+                    completion(array, nil)
+                }
+            } catch {
+                print(error)
+                completion(nil, error)
+            }
+        }
+        task.resume()
+    }
+    
+    /// Finds and parses data to extract a placeId from a set of coordinates
```

Comment:
It seems like return a set of coordinates from a given location (I may be wrong). Can you update the comment to better describe the function?

Reply:
Will do!